test/JDBC/expected/sp_tablecollations-vu-verify.out

```diff
@@ -6,3 +6,20 @@ int#!#varchar#!#binary#!#nvarchar
 ~~END~~
 
 
+-- psql
+select
```

Comment:
A small comment describing the test would be better. Also, below query looks fine better version could be not having any joins using PG's regtypes (since you anyway decided to execute as a psql query). Something like:
```
select atttypid::regtype from pg_attribute where 
   attrelid = 'sys.spt_tablecollations_view'::regclass and attname = 'name';
```